hs-init.hs

```diff
@@ -301,12 +301,18 @@ benchmarkP d =  flag Idk d
              <> help "Benchmarks"
 
 onP :: Parser Targets
-onP  = subparser $ command "on" $
-          info (helper <*> targetsP Yes) (progDesc "Specify options to enable")
+onP  = subparser $ mconcat
+       [ metavar onC
+       , command onC $ info (helper <*> targetsP Yes) (progDesc "Specify options to enable")
+       ]
+  where onC = "on [OPTIONS]"
```

Comment:
`command "on [OPTIONS]"` seems odd.  Shouldn't it remain `command "on"`?

Reply:
@Lysxia Sorry, my bad :( I was in hurry =\

Reply:
Fixed in this commit: https://github.com/vrom911/hs-init/commit/7b822eae83a3d42f9f1d2fd0467177c18bbafe9f